nj-core/src/convert.rs

```diff
@@ -126,6 +126,18 @@ impl TryIntoJs for serde_json::Value {
     }
 }
 
+#[cfg(feature = "convert-uuid")]
+impl TryIntoJs for uuid::Uuid {
+    fn try_to_js(self, js_env: &JsEnv) -> Result<napi_value, NjError> {
+        let as_str = self
+            .to_hyphenated()
+            .encode_lower(&mut uuid::Uuid::encode_buffer())
+            .to_owned();
```

Comment:
Does UUID have a parsing method such that this might be reversible? And if so, is this the "canonical" string representation for UUID? I'm wondering whether we can implement JSValue for UUID as well and want to make sure that this could be symmetrical.

Reply:
Correct, the hyphenated representation is [the canonical UUID representation](https://en.wikipedia.org/wiki/Universally_unique_identifier#Format). I would love to add a JSValue part of this, but I don't have the time at the moment, so I'll leave that bit to either future me or someone who gets to it sooner.

Reply:
Would you mind if I pushed that functionality onto your branch? I'd prefer to round this out to be feature-complete before we merge it :)

Reply:
I made a PR to this branch https://github.com/infinyon/node-bindgen/pull/162